Release/src/http/client/http_client_named_pipe.cpp

```diff
@@ -250,7 +249,7 @@ class named_pipe_client : public _http_client_communicator
 
         utility::string_t pipe_name = U("\\\\.\\pipe\\") + path[0];
 
-        while(retry_count > 0) {
+        while(retry_count-- > 0) {
```

Comment:
Can we turn this loop into a 'for' loop? I think it would make more sense in this case